cmd/manager/main.go

```diff
@@ -29,9 +29,10 @@ import (
 )
 
 const (
-	defaultLogLevel         = "info"
-	hiveNamespace           = "hive"
-	leaderElectionConfigMap = "hive-controllers-leader"
+	defaultLogLevel           = "info"
+	defaultEnableVeleroBackup = false
```

Comment:
IMO the variable name should be `enableVeleroBackup `. both `default` and `enable` sounds odd.

Reply:
Actually, this constant is never used. Thanks for pointing it out, I'll remove it.